src/main/java/com/minecolonies/coremod/colony/jobs/AbstractJob.java

```diff
@@ -312,9 +313,9 @@ public ItemStack removeItemNeeded(@NotNull final ItemStack stack)
             if (stack.isItemEqualIgnoreDurability(neededItem))
             {
                 //todo make this sofisticated as soon as material handling has been implemented.
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)